packages/react-instantsearch-hooks-web/src/__tests__/common-connector.test.tsx

```diff
@@ -311,5 +305,25 @@ createToggleRefinementConnectorTests(
       </InstantSearch>
     );
   },
-  act
-);
+};
+
+describe('Common connector tests (React InstantSearch)', () => {
+  test('has all the tests', () => {
+    expect(Object.keys(setups).sort()).toEqual(Object.keys(testSuites).sort());
+  });
+
+  const testOptions: TestOptions = {
+    createCurrentRefinementsConnectorTests: {
+      skippedTests: {
+        /** createURL uses helper state instead of ui state as it can't be translated */
+        routing: true,
+      },
+    },
+  };
+
+  Object.keys(testSuites).forEach((testName) => {
+    // @ts-ignore (typescript is only referentially typed)
+    // https://github.com/microsoft/TypeScript/issues/38520
+    testSuites[testName](setups[testName], act, testOptions[testName]);
+  });
```

Comment:
ideally this would pass typescript, but I don't find a way yet

Reply:
(note that it's not related to this PR specifically, as the ts-ignore was already here on master